proto/wasmrollapp/cron/v1beta1/cron.proto

```diff
@@ -0,0 +1,24 @@
+syntax = "proto3";
+package wasmrollapp.cron.v1beta1;
+
+import "gogoproto/gogo.proto";
+
+option go_package = "github.com/dymensionxyz/rollapp-wasm/x/cron/types";
+
+message CronJob {
+    // id is the unique identifier for the cron job
+    uint64 id = 1;
+    // name is the name of the cron job
+    string name = 2;
+    // description is the description of the cron job
+    string description = 3;
+    // Msgs that will be executed every period amount of time
+    repeated MsgContractCron msg_contract_cron = 4 [(gogoproto.nullable) = false];
```

Comment:
could we add an `enabled` field here, like suggested?

Reply:
done